src/_card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
```

Comment:
I geuss you don't have to set width of your card, paddings should define it's size